packages/o11ytsdb/src/column-store.ts

```
@@ -263,6 +295,28 @@ export class ColumnStore implements StorageBackend {
+          const cv = chunk.compressedValues;
+          const tsc = this.tsCodec;
+          const tcc = tsChunk.compressed;
+          parts.push({
+            timestamps: new BigInt64Array(0),
+            values: new Float64Array(0),
+            stats: chunk.stats,
+            chunkMinT: tsChunk.minT,
+            chunkMaxT: tsChunk.maxT,
+            decode: () => {
+              const ts = tsChunk.timestamps
+                ?? (tsChunk.timestamps = tsc ? tsc.decodeTimestamps(tcc!) : tsChunk.timestamps!);
```

Biome (2.4.11)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)